webpage/_data/navigation.yml

```diff
@@ -9,11 +9,17 @@ main:
 site_nav:
   - title: Video Tutorials
     children:
+      - title: "NANOOS Visualization System (NVS)"
+        url: https://www.youtube.com/watch?v=MEVz0jOsqmI
       - title: "IOOS conda environment"
         url: https://github.com/ioos/notebooks_demos/wiki/Installing-Conda-Python-with-the-IOOS-environment
-      - title: "OWSLib"
```

Comment:
You mean the deleted place holder links `OWSLib`, `Pyoos`, and `PyCSW`? I can try that but I never intended for them to stay, used only as place holders.